specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld

```diff
@@ -0,0 +1,79 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
```

Comment:
use the official @context

Reply:
I used the NGSI-LD example in https://github.com/GSMADeveloper/NGSI-LD-Entities/blob/master/definitions/Agri-Greenhouse.md